### PR TITLE
Change keyboardshortcut into modal

### PR DESCRIPTION
**Pull request by GretaD**
some small polishing left

- [x] Remove the client-side route that is now obsolete
- [x] Remove the server-side route and controller method that is now obsolete

### CODE REVIEW

File: src/views/KeyboardShortcuts.vue [hunk 1]
```diff
@@ -1,73 +1,77 @@
...
-		<Navigation />
-		<AppContent>
-			<AppContentDetails>
-				<h2>{{ t('mail', 'Keyboard shortcuts') }}</h2>
+	<Modal>
+		<div
+		class="modal-content"
+		size="large"
+		:title="t('mail', 'Keyboard shortcut')"
+		@close="$emit('close')">
```
Comment (skjnldsv) [inline src/views/KeyboardShortcuts.vue:7]:
```suggestion
	<Modal>
		size="large"
		:title="t('mail', 'Keyboard shortcut')"
		@close="$emit('close')">
```

Comment (ChristophWurst) [reply src/views/KeyboardShortcuts.vue:7]: :musical_note:  *if you like it then you shoulda put a ``<div/>`` on it* :musical_note:

**Comment by ChristophWurst** [comment]
Btw I think we should now put `src/views/KeyboardShortcuts.vue` to `src/components` as it's not a standalone view anymore

**Comment by GretaD** [comment]
> Btw I think we should now put `src/views/KeyboardShortcuts.vue` to `src/components` as it's not a standalone view anymore

i agree, should we include this in this PR?

**Review by ChristophWurst — APPROVED** [review]
Looks great!

**Pull request merged**